src/app/pages/search/decision-support-interventions/decision-support-interventions.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from 'react';
+
+import ChplDecisionSupportInterventionsSearchView from './decision-support-interventions-view';
+
+import { useFetchAcbs } from 'api/acbs';
+import { useFetchCriteria } from 'api/standards';
+import { FilterProvider, defaultFilter } from 'components/filter';
+import {
+  certificationBodies,
+  certificationCriteriaIds,
+  certificationDate,
+  certificationStatuses,
+} from 'components/filter/filters';
+import { AnalyticsContext, useAnalyticsContext } from 'shared/contexts';
+import { getRadioValueEntry } from 'components/filter/filters/value-entries';
+
+const staticFilters = [
+  certificationDate,
+  certificationStatuses, {
+    ...defaultFilter,
+    key: 'riskManagementSummaryInformationOptions',
+    display: 'Risk Management Summary Information',
```

Comment:
```suggestion
    display: 'RIM Summary Information',
```

```suggestion
    display: 'Risk Management Summary Info',
```

Maybe? Waiting on Jeff